doc/code_snippets/snippets/connectors/python/README.md

```diff
@@ -0,0 +1,14 @@
+# Python
+
+A sample application containing requests from the [Connecting from Python](https://www.tarantool.io/en/doc/latest/how-to/getting_started_python/) tutorial.
```

Comment:
I think it also should be explicitly clarified that the example below is for non-sharded single replicaset database, since it's a common newcomer mistake to start a sharded cluster and try to insert the data with plain `conn.insert`, expecting it to resolve sharding implicitly.

Reply:
Thanks, good point, will add this info